### PR TITLE
doc: clarify --concurrency 0

### PR DESCRIPTION
**Pull request by lambdai**
Commit Message:
Clarify doc that concurrency 0 and 1 are equivalent.

Signed-off-by: Yuchen Dai <silentdai@gmail.com>

Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/operations/cli.rst [hunk 1]
```diff
@@ -69,7 +69,8 @@ following are the command line options that Envoy supports.
 .. option:: --concurrency <integer>
 
   *(optional)* The number of :ref:`worker threads <arch_overview_threading>` to run. If not
-  specified defaults to the number of hardware threads on the machine.
+  specified defaults to the number of hardware threads on the machine. Specifically, the actual
```
Comment (alyssawilk) [inline docs/root/operations/cli.rst:72]: maybe rephrase?
"If set to zero, Envoy will still run one worker thread."

Comment (lambdai) [reply docs/root/operations/cli.rst:72]: Great! Updated
